crates/biome_js_formatter/tests/specs/prettier/typescript/import-type/import-type.ts.snap

```diff
@@ -0,0 +1,66 @@
+---
+source: crates/biome_formatter_test/src/snapshot_builder.rs
+info: typescript/import-type/import-type.ts
+---
+# Input
+
+```ts
+// ref: https://github.com/Microsoft/TypeScript/pull/22592
+
+export const x: import("./foo") = { x: 0, y: 0 };
+
+export let y: import("./foo2").Bar.I = { a: "", b: 0 };
+
+export let shim: typeof import("./foo2") = {
+    Bar: Bar2
+};
+
+export interface Foo {
+    bar: import('immutable').Map<string, int>;
+}
+
+type X = A<import("B").C<any>>;
+
+```
+
+
+# Prettier differences
+
+```diff
+--- Prettier
++++ Biome
+@@ -5,11 +5,11 @@
+ export let y: import("./foo2").Bar.I = { a: "", b: 0 };
+ 
+ export let shim: typeof import("./foo2") = {
+-  Bar: Bar2,
++    Bar: Bar2
+ };
+ 
+ export interface Foo {
+-  bar: import("immutable").Map<string, int>;
++    bar: import('immutable').Map<string, int>;
```

Comment:
This is wrong, it seems we are inserting too many spaces. Can you make sure we match prettier?

Reply:
![image](https://github.com/user-attachments/assets/a1d1be2d-27f8-4434-9448-54dacddda9bb)

this is not `space()` i add here, i am not sure why my change cause this code under the formatter wrapper by `verbatim`